src/main/java/study/view/ResultView.java

```diff
@@ -1,38 +1,55 @@
 package study.view;
 
 
+import study.core.Ladder;
 import study.core.Line;
 import study.core.PlayerName;
+import study.core.Prize;
+import study.core.Result;
 
 import java.util.List;
 
 public class ResultView {
 
-    private static final String RESULT_MESSAGE = "실행결과";
+    private static final String RESULT_MESSAGE = "실행 결과";
+    private static final String LADDER_RESULT_MESSAGE = "사다리 결과";
     private static final String LINE_HORIZONTAL = "-----";
     private static final String LINE_HORIZONTAL_EMPTY = "     ";
     private static final String LINE_VERTICAL = "|";
     private static final StringBuilder sb = new StringBuilder();
 
-    public void printLadder(List<PlayerName> names, List<Line> ladder) {
-        System.out.println(RESULT_MESSAGE);
+    public void printLadder(List<PlayerName> names, Ladder ladder, List<Prize> results) {
+        System.out.println(LADDER_RESULT_MESSAGE);
         names.forEach(name -> sb.append(String.format("%-5s", name.getName())).append(" "));
-        sb.append('\n');
-        ladder.forEach(this::printLine);
+        sb.append(System.lineSeparator());
+        ladder.getLines().forEach(this::printLine);
+        results.forEach(result -> sb.append(String.format("%-5s", result.getValue())).append(" "));
         System.out.println(sb.toString());
+        System.out.println();
     }
 
     public void printLine(Line line) {
         sb.append(LINE_HORIZONTAL_EMPTY);
         line.getPoints().forEach(point -> {
             sb.append(LINE_VERTICAL);
-            if (point) {
-                sb.append(LINE_HORIZONTAL);
-            } else {
+            if (!point) {
                 sb.append(LINE_HORIZONTAL_EMPTY);
+                return;
             }
+            sb.append(LINE_HORIZONTAL);
         });
-        sb.append(LINE_VERTICAL).append('\n');
+        sb.append(LINE_VERTICAL).append(System.lineSeparator());
+    }
+
+    public void printResult(PlayerName playerName, Result result) {
+        System.out.println();
+        System.out.println(RESULT_MESSAGE);
+        if (!playerName.getName().equals("all")) {
```

Comment:
객체에게 메시지를 보내는 방식으로 처리해보면 어떨까요?
```suggestion
        if (!playerName.isAll()) {
```